public/pages/Snapshots/containers/Snapshots/Snapshots.tsx

```diff
@@ -348,26 +346,18 @@ export default class Snapshots extends Component<SnapshotsProps, SnapshotsState>
 
     return (
       <>
-        <EuiPageHeader
-          tabs={[
-            {
-              label: "Snapshots",
-              isSelected: true,
-              onClick: this.onClickTab,
-            },
-            {
-              label: "Restore activities in progress",
-              onClick: this.onClickTab,
-            },
-          ]}
-          paddingSize="l"
-          onClick={this.onClickTab}
-        />
+        <EuiPageHeader>
+          <EuiTabs size="m" >
+            <EuiTab isSelected={true} onClick={this.onClickTab} >Snapshots</EuiTab>
+            <EuiTab onClick={this.onClickTab} >Restore activities in progress</EuiTab>
+          </EuiTabs>
+        </EuiPageHeader>
         {snapshotPanel || (
           <RestoreActivitiesPanel
             snapshotManagementService={snapshotManagementService}
-            indexService={indexService}
+            repository={selectedItems[0].repository}
```

Comment:
Are there any use cases where `selectedItems` could be an empty array? An error may occur when trying to access the `repository` attribute of the first item in an empty array.

Reply:
This is addressed in next commit.